importer/SeArbetslSv.py

```diff
@@ -136,6 +136,7 @@ def set_is(self):
         self.add_statement("is", arbetslivsmuseum, refs=[ref])
 
     def exists_with_monument_article(self, language):
+        """Override default behavior with unusual field name."""
```

Comment:
Don't we actually just set the language? Since "monument_article" is the default?